archetypes/docs/lorem/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Lorem"
```

Comment:
Let's delete this, archetypes/docs/lorem/ipsum/index.md, assets/images/default-image.png, and any empty directory with a .gitkeep file